massgov/mayflower-patternlab
# DP 8051 page banner elected org pages

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Adds a page banner variation for use on organization detail/elected official pages. 

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8051)


## Steps to Test


1. Check out `http://localhost:3000/?p=organisms-page-banner-linked`
2. Ensure other page banners are not affected




#### Impacted Areas in Application


* Added variation `@organisms/page-banner~linked`
